mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -258,3 +259,24 @@ def _delete_table_by_type_and_context_id(table_type: str, context_id: str):
             MonetDB().execute(f"DROP VIEW {name}")
         else:
             MonetDB().execute(f"DROP TABLE {name}")
+
+
+@validate_identifier_names
+def _delete_function_by_context_id(context_id: str):
```

Comment:
maybe function-> udf ?
and maybe not delete but use the sql concepts?
drop_udf_by_context_id
it describes what this function is supposed to do much better...

Reply:
actually drop_udfs_with_context_id(...)